doc/cql3/CQL.textile

```diff
@@ -414,16 +414,19 @@ bc(syntax)..
 <copy-table-stmt> ::= CREATE ( TABLE | COLUMNFAMILY ) ( IF NOT EXISTS )? <newtablename> LIKE <oldtablename>
                     ( WITH <option> ( AND <option>)* )?
 
-<option> ::= <property>
+<option> ::= <property> | INDEX
```

Comment:
@Maxwell-Guo should not it be "WITH INDEXES"? Bernardo suggested that as well below.